src/test_finder.coffee

```diff
@@ -4,19 +4,22 @@ path = require 'path'
 
 class TestFinder
 
-  constructor: ({@cwd, @testFileRegex}) ->
+  constructor: ({@cwd, @testFilePattern}) ->
 
 
   find: (done) ->
-    unless @testFileRegex
-      return done Error 'Please define a "testFileRegex" in your configuration file.'
-
-    glob(
-      "#{@cwd}/**"
-      ignore: "#{@cwd}/**/node_modules/**"
-      (err, files) =>
-        if err then return done err
-        done null, files.filter (file) => file.match @testFileRegex)
+    unless @testFilePattern
+      return done Error 'Please define a "testFilePattern" in your configuration file.'
+
+    options =
+      cwd: @cwd
+      ignore: '**/node_modules/**'
+
+    callback = (err, files) =>
+      if err then return done err
+      done null, files.map (file) => path.join(@cwd, file)
+
+    glob @testFilePattern, options, callback
```

Comment:
What is the motivation to separate the callback here? I think it would fit nicely into here if inlined.